docs/guide/content/development_guide/e2e_tests.md

```diff
@@ -0,0 +1,226 @@
+# E2E Tests
+
+OnDemand Loop uses [Cypress](https://cypress.io) for end-to-end (E2E) automated testing. These tests verify that the application works correctly from a user's perspective by simulating real interactions with the browser interface.
+
+## Overview
+
+The automated tests are located in the `e2e_tests/` directory and provide:
+
+- **End-to-end testing** of critical user workflows
+- **Docker-based execution** for consistency across development and CI/CD environments
+- **GitHub Actions integration** for automated testing on code changes
+
+### Test Structure
+
+```
+e2e_tests/
+├── cypress/
+│   ├── e2e/                    # Test specifications
+│   ├── plugins/                # Cypress plugins and utilities
+│   │   ├── config.js          # cy.loop configuration object
+│   │   ├── loop.js            # Main plugin configuration
+│   │   └── navigation.js      # Navigation utilities
+│   ├── support/               # Cypress support files
+│   │   └── e2e.js            # Global test setup
+│   └── fixtures/              # Test data files
+├── cypress.config.js          # Cypress configuration
+├── package.json              # Node.js dependencies
+├── Makefile                  # Test automation commands
+└── docker-compose.yml        # Test environment setup
+```
+
+## Running Tests Locally
+
+### Prerequisites
+
+- Docker and Docker Compose installed
+- Make utility
+- Access to test credentials (see [Local Environment](local_environment.md))
+
+### Quick Start
+
+1. **Start the test environment:**
+   ```bash
+   cd e2e_tests
+   make env_up
+   ```
+   This command starts OnDemand Loop and all required services using Docker Compose.
+
+2. **Build Cypress dependencies:**
+   ```bash
+   make cypress_build
+   ```
+
+3. **Run tests:**
+   ```bash
+   # Run all tests headless
+   make cypress_run
+   
+   # Run tests interactively (requires X11 forwarding)
+   make cypress_open
+   ```
+
+4. **Clean up:**
+   ```bash
+   make clean
+   ```
```

Comment:
Somehow `bash` modifier is not being visualized as expected

<img width="978" height="322" alt="image" src="https://github.com/user-attachments/assets/cb9e5b75-3661-4042-a9e6-276c6eb844a8" />

Reply:
Maybe you can fix it by removing the indentation

Reply:
or it can be that this markdown engine uses a different notation for code formatting